app/views/govuk_publishing_components/components/docs/step_by_step_nav.yml

```diff
@@ -417,15 +417,15 @@ examples:
 
       The solution is to track the step by step journey the user is on.
 
-      This has been achieved by adding a query string to all of the internal links in the step by step that contains the content_id of the step by step.
+      This has been achieved by adding a query string to all of the internal links in the step by step that contains the `content_id` of the step by step.
 
       For example, if the step by step contains a link to [/check-uk-visa](/check-uk-visa), the component will renders the link as [/check-uk-visa?step-by-step-nav=d8f3c2e0-d544-4664-9616-ab71323e4d18](/check-uk-visa?step-by-step-nav=d8f3c2e0-d544-4664-9616-ab71323e4d18)
```

Comment:
There's a typo in this line (not your change) that could do with correcting - `the component will renders the link` should be `the component will render the link`.